suite/runsuite_wrapper.pl

```diff
@@ -391,6 +391,7 @@
                 'prof_pcs,thread_private|common.nativeexec_bindnow_opt' => 1, # i#2052
```

Comment:
> Fixes https://github.com/DynamoRIO/dynamorio/issues/6612

No need for a new issue for something like this which targets the existing #6185 on the failing test: #6185 should be the issue in the title and description.  A PR is like an issue (same namespace) so this PR documents the act of ignoring it.

Reply:
Got it!